system/masking/include/datamaskingplugin.hpp

```diff
@@ -1929,7 +1929,7 @@ namespace DataMasking
             return false;
         }
     protected:
-        using Tester = std::function<int(int)>;
+        using Tester = int (*)(int); //std::function<int(int)>;
```

Comment:
I'll remove the commented code. Wasn't sure change would fix the Windows build break.